CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## Unreleased
+
+### Changed
```

Comment:
You should also add information regarding refactoring of the library internals.

Reply:
done